<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptingWindow</class>
 <widget class="QDialog" name="ScriptingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripting Interface</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Public Variables:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableVariables"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Loaded Scripts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listLoadedScripts"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnNewScript">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveScript">
         <property name="text">
          <string>Del</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoadScript">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,4,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Script Editing Window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnSaveScript">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRevertScript">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRecompile">
         <property name="text">
          <string>Recompile</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Log Window:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listLog"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAutoScroll">
       <property name="text">
        <string>Auto Scroll Log Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearLog">
       <property name="text">
        <string>Clear Log Window</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
